test/scalability/README.md

```diff
@@ -0,0 +1,89 @@
+# Introduction
+
+The scalability test framework attempts to determine how the product scales in terms of concurrent SQL connections
+
```

Comment:
What is it that we learn? Is it about the through-put (for which it surprises me that the data amount changes if we do inserts and selects within a workload) or is the comparison with other versions of this product?

Reply:
We want to be able to do the following two things:
- identify bottlenecks in the product, that is, workloads which do not scale well for some reason or where the performance tapers off at too small of a connection number
- evaluate suggested performance improvements against current performance, detect regressions

With respect to INSERTS + SELECTS, Alexander observed an anomaly during his benchmarking -- one possible explanation was that as you insert, the timestamp advances faster, so SELECTs have to wait less for the timestamp to advance.

Reply:
Cool! I suggest to add these two bullet points to the README.

Reply:
Thanks, will do.